app/modules.py

```diff
@@ -329,7 +360,7 @@ def __init__(self,cropFactorCollection = None):
         self.historical = HistoricalData('USA', 'AZ', datetime.datetime.today())
 
         # This value is derived from a table based on lat and lon, not calculated
-        self.dPercentofDaylight = 0.23
+        self.dPercentofDaylight = 0
```

Comment:
Is there a reason we set this to 0 instead of using setConstMeanPercentDayLight?

Reply:
because.... stuff and things

Reply:
This and the other issue have been resolved